raiden/ui/checks.py

```diff
@@ -0,0 +1,225 @@
+import sys
```

Comment:
I did not really go through he docstrings on this PR, I plan to do it in a follow up.